apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -131,7 +129,13 @@ const SwapForm = () => {
   const provider = exchangeRate?.provider;
   const providerKYC = swapKYC?.[provider];
   const kycStatus = providerKYC?.status;
-  let idleTimeout = 0;
+
+  const idleStateRef = useRef(idleState);
+  idleStateRef.current = idleState;
```

Comment:
This seems redundant with initialisation above since the "`.current` property is initialized to the passed argument `(initialValue)`" (cf. https://reactjs.org/docs/hooks-reference.html#useref)

Reply:
I have remove the 'double' initialisation although I can not remove the 'hardcode' one.
It seems there is a delay with the reference.

<img width="211" alt="Screenshot 2022-09-22 at 11 05 19" src="https://user-images.githubusercontent.com/38540290/191723694-8dc12e88-cbd4-4010-8d69-f588602eb62e.png">

Reply:
I don't really understand why we are declaring a ref `idleStateRef` since it will take the same value as `idleState` on every render.

Why not use `idleState` instead, is there something I'm missing?

Sorry don't not want to intrude on this PR but since my review was requested 😉…

Reply:
Changed.